Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'maslow'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+    }
+
+    stage('Env') {
+      govuk.setEnvar('RAILS_ENV', 'test')
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
+    }
+
+    stage('Lint') {
+      govuk.rubyLinter()
+    }
+
+    stage("Build") {
+      govuk.runRakeTask('db:mongoid:drop')
+      sh "bundle exec rake"
+    }
+
+    stage("Result") {
+      govuk.pushTag(REPOSITORY, env.BRANCH_NAME, 'release_' + env.BUILD_NUMBER)
+      govuk.deployIntegration(REPOSITORY, env.BRANCH_NAME, 'release', 'deploy')
+    }
```

Comment:
Yeah, I've been wondering about that - those tasks are conditional in the shared Groovy code, but I wonder if that's too magic. We could also duplicate that in the application Jenkinsfile if that would make people feel better.